app/i18n/locales/de/projects/proof-of-passport.json

```diff
@@ -1,4 +1,4 @@
 {
-  "description": "\nProof of passport is developing tools to let users generate proofs of humanity, nationality and age using their government-issued passport. It unlocks uses cases in proof of unique identity, sybil resistance and selective disclosure of private data.\n",
-  "tldr": "Proof of passport lets you check a passport is valid in zero-knowledge"
+  "description": "\nOpenPassport is developing tools to let users generate proofs of humanity, nationality and age using their government-issued passport. It unlocks uses cases in proof of unique identity, sybil resistance and selective disclosure of private data.\n",
```

Comment:
"n"OpenPassport 
is it on purpose or? the n

Reply:
The "\n" is a common way to tell parsers to insert a new line. I'm not sure why this needed here in that case. Also, I can't find any other projects in these folders that use this technique.